api/v1/usage_server.proto

```diff
@@ -28,6 +28,56 @@ message AggregatedSummary {
   repeated Summary method_summaries = 2;
 }
 
+message GetUsageDataRequest {
+  string tenant_id = 1;
+
+  // start_time is the UNIX timestamp for the data start time (inclusive).
+  // If start_time is not provided, the default is the 24 hours before end_time.
+  int64 start_time = 2;
+  // end_time is the UNIX timestamp for the data end time (exclusive).
+  // If end_time is not provided, the default is the current time.
+  int64 end_time = 3;
+
+  Filter filter = 4;
+
+  // after is the identifier for the last datum from the previous pagination request.
+  string after = 5;
+  // limit is the number of entries to retrieve.
+  int32 limit = 6;
+
+  message Filter {
+    repeated string api_keys = 1;
+    repeated string models = 2;
+  }
+}
+
+
+
+message UsageData {
+  string method = 1;
```

Comment:
Could you explain what this field means?

Reply:
removed it, I think it snuck in with co-pilot